codegen/src/generator/tests.rs

```diff
@@ -16,10 +16,10 @@ const OUR_TESTS_DIR: &'static str = concat!(env!("CARGO_MANIFEST_DIR"), "/tests/
 pub fn generate_tests(writer: &mut FileWriter, service: &Service) -> IoResult {
     writeln!(writer,
              "
-            #[cfg(test)]
-            mod protocol_tests {{
-                {tests_body}
-            }}
+#[cfg(test)]
```

Comment:
Per Slack convo, shall we wait for the subsequent PR that is only rustfmt changes?

I agree with your position, the part humans read, the codegen is probably better to make more readable.